samples/shields/npm1300_ek/prj.conf

```diff
@@ -3,3 +3,4 @@
 
 CONFIG_SHELL=y
 CONFIG_LOG=y
+CONFIG_REGULATOR=y
```

Comment:
missing gpio and sensor here

Reply:
`CONFIG_SENSOR=y` added, but I omit GPIO, which is part of default boards configuration (so theoretically should always be there).
But I can add it, if you think there's a risk it is finally missing.